configs/zr/playerclass.json.example

```diff
@@ -0,0 +1,118 @@
+// nlohmann json parser allows for use of // and /**/ comments by setting `ignore_comments` parameter to true in parsing function
+{
+	"Human":
+	{
+		"HumanClass1":
+		{
+			// class behavior, values will not be inherited from base class
+			"enabled":					true,
+			"team_default":				true,
+
+			// player behavior
+			// all player attributes are required if it doesn't have a base class
+			"health":					100,
+			"model":
+			[
+				{
+					"modelname":		"characters/models/ctm_fbi/ctm_fbi.vmdl", // required, otherwise model entry is skipped
+					"color":			"255 255 255", // optional, defaults to "255 255 255"
+					"skin":				0 // optional, can be a single integer or array of integers. Defaults to 0
```

Comment:
Imo we should use `models` and `skins` just to emphasize the available functionality.